### PR TITLE
W: [fr] Software Manual

### PR DESCRIPTION
**Pull request by gegeweb**
/ping @trebmuh @Saimyx

### CODE REVIEW

**Comment by gegeweb** [comment]
The note about the definition of "gigue" is not in the English version.
In French "gigue" is also a dance and is a term probably not known to most musicians.
I discovered its meaning by translating this page!

**Comment by ann0see** [comment]
This page will probably be moved and restructured quite a bit in future.

**Comment by gegeweb** [comment]
> This page will probably be moved and restructured quite a bit in future.

Ok, i'll not spend time with localized screenshot for the moment.

Since last commit (some rewriting), ready for review.

File: wiki/fr/fr-Software-Manual.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+layout: wiki
+title: "Manuel du Logiciel"
+lang: "fr"
+permalink: "/wiki/Software-Manual"
+---
+Aide Jamulus (Manuel du Logiciel)
+=================================
+_(Note: les copies d'écran ont été faite avec l'interface en anglais, les termes utilisés dans ce manuel correspondent aux termes employés dans l'interface du logiciel en français.)_
```
Comment (ann0see) [inline wiki/fr/fr-Software-Manual.md:9]: Thanks for adding this!

**Review by ann0see — APPROVED** [review]

**Comment by gegeweb** [comment]
About "mute", when active: « en sourdine » ou « en silence »?  I think « en silence » should be ok with « Me silencer » (sounds strange for me).
I need to re-read again before reviewing. I'm not happy with all.

**Comment by gegeweb** [comment]
I've used here terms used by the French sound technicians/engineers.

- « Console de mixage » instead of « mixeur, mélangeur or table de mixage (DJ) »
- « Potentiomètres [linéaires] » for « fader » even if « faders » or « fadeurs » is used orally by by the corporation. This two terms exists in French with other définitions. « Potentiomiètres » is the good terms on a mixing desk (comes from the name of the electronic component).
- On mixing desk a channel is « une tranche » in French, but « canal » is used in the French UI and make more sens for the final user.

Hope I've not (I'll check again) used « potentiomètre » where « curseur » must be used and i've made a good usage of « tampons » and « échantillons ».

Ready for review (bis).

**Pull request closed without merging**